src/MensaWidget/MWFrame.class.st

```diff
@@ -11,43 +11,160 @@ Class {
 	#'squeak_changestamp' : 'Louis Christ 5/27/2024 19:43'
 }
 
+{
+	#category : #constants,
+	#'squeak_changestamp' : 'tmc 6/6/2024 21:40'
+}
+MWFrame class >> cellSpacing [
+
+	^ 10
+]
+
+{
+	#category : #constants,
+	#'squeak_changestamp' : 'tmc 6/6/2024 21:36'
+}
+MWFrame class >> contentExtent [
+
+	^ 450 @ 550
+]
+
+{
+	#category : #constants,
+	#'squeak_changestamp' : 'tmc 6/6/2024 21:03'
+}
+MWFrame class >> contentMorphColor [
+
+	^ Color darkGray
+]
+
+{
+	#category : #constants,
+	#'squeak_changestamp' : 'Oscar 6/5/2024 00:54'
+}
+MWFrame class >> contentMorphInnerBorder [
+
+	^ 5
+]
+
+{
+	#category : #constants,
+	#'squeak_changestamp' : 'tmc 6/6/2024 21:50'
+}
+MWFrame class >> contentPosition [
+
+	^ 5 @ 40
+]
+
+{
+	#category : #constants,
+	#'squeak_changestamp' : 'Louis Christ 6/6/2024 19:50'
+}
+MWFrame class >> frameSize [
+
+	^ 460 @ 595
+]
+
+{
+	#category : #constants,
+	#'squeak_changestamp' : 'Oscar 6/5/2024 01:01'
+}
+MWFrame class >> scrollBarThickness [
+
+	^ 20
+]
+
+{
+	#category : #constants,
+	#'squeak_changestamp' : 'Oscar 6/5/2024 00:57'
+}
+MWFrame class >> spacingBetweenCanteenMorphs [
```

Comment:
I cannot find any "sender" of the variable. Is the variable unused, or did I overlook something?